src/Illuminate/Foundation/Http/Middleware/Authorize.php

```diff
@@ -38,6 +38,8 @@ public function __construct(Gate $gate)
      */
     public function handle($request, Closure $next, $ability, $model = null)
     {
+        auth()->authenticate();
```

Comment:
Should probably avoid calling the `auth()` method here since we're already bothering to use DI to inject the gate.